docs/src/index.md

```diff
@@ -4,7 +4,7 @@ DocTestSetup = :(using Snowflurry)
 ```
 *A library for quantum computing using Julia*
 
-Snowflurry is a pure Julia quantum computing stack that allows you to easily design quantum circuits, experiments and algorithms. Snowflurry can run these quantum applications on real quantum computers or classical simulators.
```

Comment:
The use of the term "quantum computing stack" to describe Julia is inaccurate since the quantum computing stack typically includes the quantum processor as well. I have used the term "software package" instead. I am open to other suggestions.

Reply:
I'm happy with software package. Can't think of anything better either.